tests/Makefile

```diff
@@ -77,7 +77,7 @@ lint:
 	$(V)$(HELM) lint ../kubernetes/zenko/charts/* && $(HELM) lint ../kubernetes/zenko/
 
 build-e2e:
-	$(V)$(DOCKER) build -t $(E2E_IMAGE) . && \
+	$(V)$(DOCKER) build -t $(E2E_IMAGE) python_tests && \
```

Comment:
IMHO, there is no need to add the suffix `_tests` to python if this is already under the tests directory. Also, it doesn't look good to define a test section by the name of a language.

Reply:
what would you propose?

Reply:
It's really hard to say due to the current state of our not-so-well-defined tests suite.